source/includes/fact-verifier-limitations.rst

```diff
@@ -0,0 +1,40 @@
+
+Compatibility
+~~~~~~~~~~~~~
+
+The embedded verifier is not available in mongosync 1.8 and
+earlier.
+
+For alternative verification methods, see :ref:`c2c-verification`.
+
+Limitations
+~~~~~~~~~~~
+
+The embedded verifier has the following limitations:
+
+- ``mongosync`` stores the verifier state in memory, which can
+  result in a significant memory overhead. To run verification
+  checks, ``mongosync`` requires approximately 1 GB of memory
```

Comment:
Can we  update this to 0.5GB per 1 million documents?

Reply:
Fixed